src/main/java/com/mongodb/spark/sql/connector/exceptions/DataException.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.mongodb.spark.sql.connector.exceptions;
+
+/** A base class for all data exceptions. */
+public class DataException extends RuntimeException {
```

Comment:
Done

Reply:
Just in case, setting `serialVersionUID` to 0 or 1 (the way we usually start versioning) appears more natural than using an essentially random number.

My understanding is that initially the intent of the language designers was to generate an implicit `serialVersionUID` automatically (this still happens, unfortunately), so that when the serial form of a class changes as a result of user changing the class, the `serialVersionUID` is updated automatically. Unfortunately, the specification itself has wording that renders this idea useless: [_"Synthetic fields generated by javac (or other JavaTM compilers) to implement inner classes are implementation dependent and may vary between compilers; differences in such fields can disrupt compatibility as well as result in conflicting default serialVersionUID values."_](https://docs.oracle.com/en/java/javase/16/docs/specs/serialization/serial-arch.html#the-serializable-interface).

Reply:
Done!